src/extension/content-script/tasks.ts

```diff
@@ -67,6 +67,57 @@ export default function tasks(...args: Parameters<typeof realTasks>) {
     // for debug purpose
     return _tasks
 }
+
+const uriCanDoTask = (tabUri?: string, targetUri?: any) => {
+    if (tabUri?.startsWith(targetUri)) return true
+    return false
+}
+
+/**
+ * ! For mobile standalone app
+ * This function will open/switch tabs (and start tasks)
+ * It tries to check if a tab being able to do the target task already opened before opening it
+ * For Chrome, `browser.tabs.query` won't work for chrome-extension: schema, so this function is not useful
+ */
+
+export function exclusiveTasks(...args: Parameters<typeof realTasks>) {
```

Comment:
Why not merge this method into `tasks`? No need for the caller knowing about the current build target.

Reply:
the word `exclusive` means its target URL is exclusive on mobile... for example, we should not open another tab if there is already one can do the task. we don't utilize this feature on the extension version now.

merge this into `tasks` is ok too but it seems not offering any benefit.